clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -490,6 +500,24 @@ public void ingestSSTFiles(RocksDB rocksDB, List<ColumnFamilyHandle> columnFamil
     }
   }
 
+  public void createSnapshot(RocksDB rocksDB) {
+    try {
+      Checkpoint checkpoint = makeCheckpoint(rocksDB);
+      String checkpointPath = this.fullPathForPartitionDBSnapshot;
+
+      if (Files.exists(Paths.get(checkpointPath))) {
+        return;
+      }
+
+      LOGGER.info("Start creating snapshots in directory: {}", checkpointPath);
+      checkpoint.createCheckpoint(checkpointPath);
+
+      LOGGER.info("Finished creating snapshots in directory: {}", checkpointPath);
+    } catch (RocksDBException e) { // | IOException e) {
+      throw new VeniceException("Received exception during RocksDB's snapshot creation", e);
```

Comment:
Can you please add a little more details in the exception? Like what's the path so it's easier to debug using the exception msg. (IDK if `e` contains it or not. If yes, feel free to discard this comment)